lms/djangoapps/completion/waffle.py

```diff
@@ -86,6 +90,18 @@ def visual_progress_enabled(course_key):
     return True
 
 
+def new_course_outline_enabled(course_key):
+    """
+    Returns whether the new course outline is enabled,
+    which requires visual progress to be enabled, as well.
+
+    :return:
+        bool -> True if visual progress is enabled
+        and the new course outline is enabled
+    """
+    return waffle().is_enabled(ENABLE_NEW_COURSE_OUTLINE) and visual_progress_enabled(course_key)
```

Comment:
If you setup the flag this way, there is no good way to enable the new course outline course by course without affecting the visual progress features.
What you should do is to create two new course outline flags, one for course override, one for all of them. it should be totally independent from visual_progress_enabled flag.

Reply:
...and a key for site configuration, too.  Also, since this doesn't really have anything to do with completion, can we make a `openedx.features.course_experience.waffle` file and add all that stuff there?